src/renderers/webgl/WebGLTextures.js

```diff
@@ -1133,6 +1133,8 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 				_gl.blitFramebuffer( 0, 0, width, height, 0, 0, width, height, mask, _gl.NEAREST );
 
+				_gl.bindFramebuffer( _gl.FRAMEBUFFER, renderTargetProperties.__webglMultisampledFramebuffer );
```

Comment:
Good catch! Do you mind adding the issue as a comment like so:
```js
_gl.bindFramebuffer( _gl.FRAMEBUFFER, renderTargetProperties.__webglMultisampledFramebuffer ); // see #18905
```
This makes the line more clear for devs who aren't familiar with the respective history of this change.